src/platform/apollolake/include/platform/clk-map.h

```diff
@@ -33,19 +33,26 @@
 
 #include <sof/clk.h>
 
+#define CLK_MAX_CPU_HZ		400000000
+#define CLK_DEFAULT_CPU_HZ	CLK_MAX_CPU_HZ
```

Comment:
Can we get rid of `CLK_DEFAULT_CPU_HZ` at all? It's not used on cavs platforms.

Reply:
Removed